accelerator/config.c

```diff
@@ -175,8 +175,8 @@ status_t ta_core_default_init(ta_core_t* const core) {
   ta_log_info("Initializing DB connection\n");
   db_service->host = strdup(DB_HOST);
 #endif
-  // Turn off verbose mode default
-  verbose_mode = false;
+  // Turn off quiet mode default
```

Comment:
They are opposite! What is the default value?

Reply:
There are two mode here
1. verbose mode: Print all the message above info level
2. quiet mode: Only print with method has been called by client.

The default mode was quiet mode. And now I change the default mode to verbose mode. If the user don't want to see to many information, then they should run in quiet mode.